fetch_response.go

```diff
@@ -385,6 +385,65 @@ func (r *FetchResponse) AddRecordWithTimestamp(topic string, partition int32, ke
 	batch.addRecord(rec)
 }
 
+// AddRecordBatchWithTimestamp is similar to AddRecordWithTimestamp
+// But instead of append a record a batch of record it append a new batch of record of size 1 to a set of batch
```

Comment:
```suggestion
// But instead of appending .... 
```
Here I'm not sure to fully understand your comment. Could we rephrase it?

Reply:
how is it now?